content/shmem_wait.tex

```diff
@@ -45,14 +45,16 @@
 
 \apidescription{ 
     \FUNC{shmem\_wait} and \FUNC{shmem\_wait\_until} wait for \VAR{ivar} to be
-    changed by a remote write or an atomic operation issued by a different \ac{PE}.
+    changed by a write or an atomic operation issued by a
+    \oldtext{different} \ac{PE}.
     These  routines can be used for point-to-point direct synchronization.  A call
-    to \VAR{shmem\_wait} does not return until some other \ac{PE} writes  a value,
+    to \VAR{shmem\_wait} does not return until a \ac{PE} writes  a value,
```

Comment:
This was unchanged because it was from the old text. Since it's only a latex fix, I will fix it, if there are no objections.